### PR TITLE
Correction to web script

### PR DESCRIPTION
**Pull request by alunturner**
Adds correction to the `update` script in package.json.

The script automates building the bindings, copying the files, then rebuilding the web code.

### CODE REVIEW

**Review by aringenbach — APPROVED** [review]
LGTM

File: platforms/web/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
         "test:watch": "vitest",
         "test:ui": "vitest --ui",
         "coverage": "vitest run --coverage",
-        "update": "cd ../../bindings/wysiwyg-wasm && npm run build && cp -f ./pkg/*wysiwyg* ../../platforms/web/generated && cd ../../platforms/web && yarn"
+        "update": "cd ../../bindings/wysiwyg-wasm && npm run build && cp -f ./pkg/*wysiwyg* ../../platforms/web/generated && cd ../../platforms/web && yarn build"
```
Comment (florianduros) [inline platforms/web/package.json:40]: We should yarn instead of npm to run script since we are using yarn in the RTL

Comment (alunturner) [reply platforms/web/package.json:40]: I agree that we are in /platforms/web, but it doesn't appear to me that we are inside the /bindings/wysiwyg-wasm folder. It also says in the readme (in that folder) that `npm run build` is what to run there.

Am I missing something here? Not sure if the README should be updated or followed.

Comment (florianduros) [reply platforms/web/package.json:40]: Oh you're right, npm is used in the `wysiwyg-wasm`folder ! 

You can ignore my comment

**Pull request merged**